eng/Version.Details.xml

```diff
@@ -242,5 +242,10 @@
       <Sha>4234ffac87e305da80597cb46dc0d4a87fc4f7c2</Sha>
       <SourceBuild RepoName="xliff-tasks" ManagedOnly="true" />
     </Dependency>
+    <Dependency Name="runtime.linux-x64.Microsoft.NETCore.Runtime.ObjWriter" Version="8.0.0-preview.6.23316.4" CoherentParentDependency="Microsoft.NETCore.App.Runtime.win-x64">
```

Comment:
Ideally this would be defined within the runtime's version.details.xml.  Additionally the use of SourceBuildTarball should be updated to just SourceBuild.  I am trying to remove the SourceBuildTarball attribute.

Reply:
I copied it from the old PR. I will update it.

I have no idea what this does. Can you give a short description how this is used, or point me to some doc?

Reply:
> Ideally this would be defined within the runtime's version.details.xml.

You want me to make a PR against https://github.com/dotnet/runtime?
and add
```

    <Dependency Name="runtime.linux-x64.Microsoft.NETCore.Runtime.ObjWriter" Version="8.0.0-preview.6.23316.4" CoherentParentDependency="Microsoft.NETCore.App.Runtime.win-x64">
      <Uri>https://github.com/dotnet/llvm-project</Uri>
      <Sha>30e9e6bc2e9f04e0a75daf4b8088ee91f66069da</Sha>
    </Dependency>
```
to its `eng/Version.Details.xml`?

Reply:
It looks like these are already defined:

https://github.com/dotnet/runtime/blob/178425101dd6f697682031a9a953450d82719eb6/eng/Version.Details.xml#L15-L46

Reply:
> Can you give a short description how this is used

It does two things.

1. It indicates that the dependency should be included in the VMR.  It specified the digest to include.  The version details graph is walked and the first entry wins if there are multiple for a repo.

2. It tells the source build infra to pull in the intermediate package for the repo when building the repo level source build.  The intermediate contents pre-populate the local nuget cache for prebuilt detection.

> It looks like these are already defined:

One will need the <SourceBuild> element added to it.

Reply:
> One will need the element added to it.

I don't understand what you mean.

Reply:
Apologies, I used angle brackets in my original response and it wasn't rendered.  I updated my comment.